app/webpacker/components/wca/FormBuilder/input/FormInputs.js

```diff
@@ -206,14 +206,14 @@ export const InputNumber = wrapInput((props) => {
       name={props.htmlName}
       type="number"
       label={props.attachedLabel}
-      value={props.value}
+      value={props.value ?? ''}
```

Comment:
(Didn't run the code locally, so may be painfully obvious) Why is this `??` fallback necessary?

Reply:
Oop it isn't - just a holdover from when I wasn't doing things properly on 199 - removed